### ISSUE TITLE
第四章 nn.Sequential(*layers)为什么不是nn.ModuleList(layers)?

### ISSUE THREAD

**Issue by XiumeiChen** [chenyuntc/pytorch-book#110]
def _make_layer(self,  inchannel, outchannel, block_num, stride=1):
        '''
        构建layer,包含多个residual block
        '''
        shortcut = nn.Sequential(
                nn.Conv2d(inchannel,outchannel,1,stride, bias=False),
                nn.BatchNorm2d(outchannel))
        
        layers = []
        layers.append(ResidualBlock(inchannel, outchannel, stride, shortcut))
        
        for i in range(1, block_num):
            layers.append(ResidualBlock(outchannel, outchannel))
        return nn.Sequential(*layers)

你好，最后一句的Sequential()的用法不是前文所列的三种用法之一，倒像是Modellist的用法，为什么呢？

**Comment by Diamondfan** [comment]
@XiumeiChen  Modulelist没有定义前向计算，类似于list将Module放在一起。Sequential是定义了前向计算过程的。

**Comment by XiumeiChen** [comment]
> @XiumeiChen Modulelist没有定义前向计算，类似于list将Module放在一起。Sequential是定义了前向计算过程的。

明白了，谢谢。
那为什么要加星号？return nn.Sequential(*layers)而不是return nn.Sequential(layers)？

**Comment by XiumeiChen** [comment]
@Diamondfan 明白了，谢谢。
那为什么要加星号？return nn.Sequential(*layers)而不是return nn.Sequential(layers)？
